addons/project_todo/static/tests/todo_conversion_form_view.test.js

```diff
@@ -62,6 +62,7 @@ test("Check that todo_conversion_form view focuses on the focus on the first ele
         resId: 1,
         type: "form",
         actionMenus: {},
+        viewId: 1,
```

Comment:
is that needed? :thinking: shouldn't specifying "form" in the type suffice? 
the same applies to the other test ``todo_form_view.test.js``

Reply:
Here we need to open the particular view that's why we needed viewId.